### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ReDeiPirati**
Add Encoding issue for In-app alerts and warn users about serving supported envs

### CODE REVIEW

File: docs/commands/run.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ floyd run [OPTIONS] [COMMAND]
 | command |    | Command to execute when running your project on Floyd. |
 | command |      | Command to execute when running your project on Floyd. |
 
+!!! warning "Encoding issue!"
+    _This is a common issue between **Windows users**_: it could happens that your Terminal is using a different encoding from the one expected from the remote machine. Here are some examples:
```
Comment (whatrocks) [inline docs/commands/run.md]: _This is a common issue for **Windows users**_: This occurs when your Windows terminal is using a different encoding from the one expected by the remote machine. Here are some examples:

File: docs/commands/run.md [hunk 2]
```diff
@@ -18,6 +18,15 @@ floyd run [OPTIONS] [COMMAND]
...
+    
+    1. Slash and single quote issue: `floyd run \ --data alice/datasets/test \  'python test_Sony.py'` is translated as `floyd run --data alice/datasets/test '\ \ \ \ '"'"'python test_Sony.py'"'"''`  
+    2. Double quotes issue: `floyd run "python test_Sony.py"` is translated as `floyd run  ''"'"'python test_Sony.py'"'"''`
+
+    Unfortunately, there isn't a silver bullet, but you can use the `Command` view of the Job's Overview page to help at debugging it. More in general, if you will notice this issue, you can try to switch single quotes with double or viceversa, and remove the slash if you are indenting the commands on multiple lines - these usually fix the issue in 99% of the case. If it will not be the case, please reach out us at support@floydhub.com.
```
Comment (whatrocks) [inline docs/commands/run.md]: Unfortunately, there isn't a silver bullet to identifying this issue, but you can use the `Command` view of the Job's Overview page to help debug and identify this issue. More generally, if you notice this issue, you can try to switch single quotes with double or vice versa, and remove the slash if you are indenting the commands on multiple lines - these changes usually fix the issue in 99% of the cases. If not, please reach out to us at support@floydhub.com.

File: docs/faqs/using_floydhub.md [hunk 3]
```diff
@@ -86,3 +86,12 @@ if os.environ.get('FLOYDHUB'):
 else:
 	# Running locally or in other platforms
 ```
+
+### Encoding issue
```
Comment (whatrocks) [inline docs/faqs/using_floydhub.md:90]: same changes as above

File: docs/faqs/using_floydhub.md [hunk 4]
```diff
@@ -86,3 +86,12 @@ if os.environ.get('FLOYDHUB'):
...
+
+_This is a common issue between **Windows users**_: it could happens that your Terminal is using a different encoding from the one expected from the remote machine. Here are some examples:
+    
+1. Slash and single quote issue: `floyd run \ --data alice/datasets/test \  'python test_Sony.py'` is translated as `floyd run --data alice/datasets/test '\ \ \ \ '"'"'python test_Sony.py'"'"''`  
+2. Double quotes issue: `floyd run "python test_Sony.py"` is translated as `floyd run  ''"'"'python test_Sony.py'"'"''`
+
+ Unfortunately, there isn't a silver bullet, but you can use the `Command` view of the Job's Overview page to help at debugging it. More in general, if you will notice this issue, you can try to switch single quotes with double or viceversa, and remove the slash if you are indenting the commands on multiple lines - these usually fix the issue in 99% of the case. If it will not be the case, please reach out us at support@floydhub.com.
```
Comment (whatrocks) [inline docs/faqs/using_floydhub.md:97]: same as above

File: docs/guides/serving.md [hunk 5]
```diff
@@ -52,7 +52,13 @@ if __name__ == "__main__":
 You can also view a complete example [here](https://github.com/floydhub/fast-style-transfer/blob/master/app.py).
 
 !!! important "Pay attention to Race Conditions!" 
-    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving a model is considered not recommended, but if your use-case requires it, remember to lock the disk resources every time you write to them. 
+    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving model is considered an anti-pattern, but if your use-case require it, please remember to lock the disk resources every time you will need to write and read from them.
```
Comment (whatrocks) [inline docs/guides/serving.md]: but if your use-case requires it

File: docs/guides/serving.md [hunk 6]
```diff
@@ -52,7 +52,13 @@ if __name__ == "__main__":
 You can also view a complete example [here](https://github.com/floydhub/fast-style-transfer/blob/master/app.py).
 
 !!! important "Pay attention to Race Conditions!" 
-    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving a model is considered not recommended, but if your use-case requires it, remember to lock the disk resources every time you write to them. 
+    The uswgi application server will spawn 2 workers (processes) for serving your application. This can be a source of [race conditions](https://en.wikipedia.org/wiki/Race_condition#software)! Writing on disk when serving model is considered an anti-pattern, but if your use-case require it, please remember to lock the disk resources every time you will need to write and read from them. 
+
+!!! warning "Environments available for serving mode" 
+    Serving is available only for these environments:
+    
+    - TensoFlow (`--env tensorlfow-1.5` and later)
```
Comment (whatrocks) [inline docs/guides/serving.md]: TensorFlow

Comment (whatrocks) [reply docs/guides/serving.md]: tensorflow-1.5`

**Review by whatrocks — CHANGES_REQUESTED** [review]
Looks good - just a few spelling updates.

**Comment by ReDeiPirati** [comment]
Thanks @whatrocks!

**Comment by ReDeiPirati** [comment]
@narenst @houqp quick review, please?

**Review by whatrocks — APPROVED** [review]

**Pull request merged**